tests/test_mask.py

```diff
@@ -71,14 +87,51 @@ def test_estimate_missing_mask(self):
         """
         Test mask estimations without mask on the face
         """
-        expectedResult = MaskProperties(0.896, 0.078, 0.024)
+        expectedResult = MaskProperties(0.998, 0.001, 0.000)
         mask = TestMask.maskEstimator.estimate(self.warpImageMissing)
         self.assertMaskEstimation(mask, expectedResult._asdict())
 
     def test_estimate_mask_occluded(self):
         """
         Test mask estimations with face is occluded by other object
         """
-        expectedResult = MaskProperties(0.326, 0.142, 0.531)
+        expectedResult = MaskProperties(0.027, 0.097, 0.875)
         mask = TestMask.maskEstimator.estimate(self.warpImageOccluded)
         self.assertMaskEstimation(mask, expectedResult._asdict())
+
+    def test_estimate_maskV2_as_dict(self):
+        """
+        Test maskV2 estimations as dict
+        """
+        faceDetection = self.defaultDetector.detectOne(self.imageMedicalMask)
+        maskDict = TestMask.maskEstimator.estimate(self.imageMedicalMask, faceDetection.detection).asDict()
+        assert (
+            jsonValidator(schema=MASK_SCHEMA).validate(maskDict) is None
+        ), f"{maskDict} does not match with schema {MASK_SCHEMA}"
+
+    def test_estimate_medical_mask_v2(self):
```

Comment:
it should be subtest of `test_estimate_medical_mask`

Reply:
Done